.github/file-paths.yaml

```diff
@@ -0,0 +1,70 @@
+shared_sources: &shared_sources
+  - "shared/src"
+
+shared_specs: &shared_specs
+  - "shared/test"
+
+frontend_sources: &frontend_sources
+  - *shared_sources
+  - "frontend/**"
+  - "enterprise/frontend/**"
+  - "frontend/!(test)/**"
+  - "yarn.lock"
+  - "**/tsconfig*.json"
+  - "package.json"
+  - "babel.config.json"
+  - "postcss.config.js"
+  - "webpack.config.js"
+  - "webpack.static-viz.config.js"
+
+frontend_specs: &frontend_specs
+  - *shared_specs
+  - "frontend/test/!(__support__|__runner__)/**"
+  - "frontend/**.unit.*"
+  - "jest.config.js"
+
+frontend_all: &frontend_all
+  - *frontend_sources
+  - *frontend_specs
+
+backend_presto_kerberos:
+  - "**/presto_jdbc/**"
+  - "**/presto_jdbc.clj"
+
+backend_sources: &backend_sources
+  - *shared_sources
+  - "**deps.edn"
+  - "enterprise/backend/src/**"
+  - "src/**"
+  - "*modules/**"
```

Comment:
Shouldn't this be `modules/**`? What's the significance of the star at the front?

Reply:
Will match `modules/**` and `test_modules/**`

Reply:
Ah, I wasn't aware of `test_modules`. Seems like this is needlessly general though. Can we list these separately?

Reply:
if you do modules/**, it'll also match `modules/redshift/test` which includes driver tests

Reply:
sorry if I haven't made myself clear: not everything in `modules/redshift/test` folder are test files.

for example `modules/drivers/bigquery-cloud-sdk/test/metabase/test/data/bigquery_cloud_sdk.clj` is a source file, we define test utils there.

But `modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj` is a test file.

Reply:
@calherries @qnkhuat thanks for your findings, I updated the paths and I decided to exclude BE and e2e specs from this PR to test everything on the FE jobs.

BE workflows are changed here https://github.com/metabase/metabase/pull/28171